tests/ci/config.go

```diff
@@ -0,0 +1,51 @@
+package ci
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"strings"
+
+	CON "github.com/terraform-redhat/terraform-provider-rhcs/tests/utils/constants"
+)
+
+var RHCS = new(RHCSconfig)
+
+// RHCSConfig contains platforms info for the RHCS testing
+type RHCSconfig struct {
+	// Env is the OpenShift Cluster Management environment used to provision clusters.
+	RHCSEnv           string `env:"RHCS_ENV" default:"staging" yaml:"env"`
+	ClusterProfile    string `env:"CLUSTER_PROFILE" yaml:"clusterProfile,omitempty"`
+	ClusterProfileDir string `env:"CLUSTER_PROFILE_DIR" yaml:"clusterProfileDir,omitempty"`
+
+	YAMLProfilesDir string
+}
+
+func init() {
+	currentDir, _ := os.Getwd()
+	project := "terraform-provider-rhcs"
+	rootDir := GetEnvWithDefault(CON.WorkSpace, strings.SplitAfter(currentDir, project)[0])
+
+	RHCS.RHCSEnv = "staging"
```

Comment:
Why do you set it here hardcoded? It is the same with default value

Reply:
fixed